docs/quickstarts/openshift-add-cluster-user/metadata.yml

```
@@ -1,5 +1,5 @@
 kind: QuickStarts
-name: rosa-add-cluster-user
+name: OpenShift-add-cluster-user
```

```suggestion
name: rosa-osd-add-cluster-user
```
Suggest keeping the same naming convention that we've used for the other updated quickstarts.